README.md

```diff
@@ -1,30 +1,26 @@
 [![SWUbanner](https://raw.githubusercontent.com/vshymanskyy/StandWithUkraine/main/banner2-direct.svg)](https://supportukrainenow.org/)
 
-# Microservice template for modern PHP applications
+# User Service
+
 
-[![CodeScene Code Health](https://codescene.io/projects/39797/status-badges/code-health)](https://codescene.io/projects/39797)
```

Comment:
We should connect and add the right badges from the codescene instead of removing it
Please ask in backend channel for setting up the codescene